hadoop-ozone/dist/src/main/smoketest/acommonlib.robot

```diff
@@ -0,0 +1,20 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
What is the goal of this file? Based on the name it's not clear for me.

Reply:
Oh, I got it. I think you assume it's always executed before all the tests. I would use a generic test initialization file:

http://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#initialization-files

and a 'Suite Setup'. Kinit can be executed in case of the SECURITY_ENABLED is set. An other init file can be added to the s3 directory to initialize the s3 credentials.

Reply:
done